src/containers/Review/assignment/AssignmentTab.tsx

```diff
@@ -118,7 +117,6 @@ const AssignmentTab: React.FC<{
                   sectionId={id}
                   fullStructure={fullStructure}
                   reviewAssignment={assignment}
-                  previousAssignment={assignmentInPreviousStage}
```

Comment:
Can you just quickly remind me why we're not using this anymore?

Reply:
It wasn't been used. It was a optional parameter passed along to `ReviewSectionRow` to `useCreateFinalDecisionReview` and `useGetReviewStructureForSectionAsync ` to the helpers which should include that on the `compileVariablesForReviewResponseQuery` that didn't use it, if you check this call:
https://github.com/openmsupply/conforma-web-app/blob/59cf933cb71e72059ab554e0d2c5269c8aa3b31f/src/utils/hooks/useGetReviewStructureForSection/helpers.tsx#L54-L72

Very difficult to track it down, but in the end `previousAssignment` was already being ignored! That's a previous change that missed the tidy up part, the previous Stage responses are fetched in `getReviewResponses` without the need to pass the previousAssignment.